src/componentsNext/Filepicker/index.tsx

```diff
@@ -0,0 +1,146 @@
+import { Breadcrumb, BreadcrumbRoot } from "~/design/Breadcrumbs";
+import {
+  Filepicker,
+  FilepickerClose,
+  FilepickerHeading,
+  FilepickerList,
+  FilepickerListItem,
+  FilepickerSeparator,
+} from "~/design/Filepicker";
+import { FolderUp, Home } from "lucide-react";
+import { Fragment, useState } from "react";
+
+import Button from "~/design/Button";
+import { ButtonBar } from "~/design/ButtonBar";
+import Input from "~/design/Input";
+import { analyzePath } from "~/util/router/utils";
+import { fileTypeToIcon } from "~/api/tree/utils";
+import { useNodeContent } from "~/api/tree/query/node";
+import { useTranslation } from "react-i18next";
+
+const FilepickerMenu = ({
+  namespace,
+  defaultPath,
+  onChange,
+}: {
+  namespace: string;
+  defaultPath?: string;
+  onChange: (newValue: string) => void;
```

Comment:
Maybe `newValue` could be named `filePath`? Just to make it a bit more clear, what the string actually contains? 

I would also suggest making the `onChange` optional, even if the user might always need one, I like to have the onChange match the DOM elements behavior. The same way you can also place an input field somewhere and don't do anything with the user's input.

You typed the namespace as required but also implemented a backup if the namespace is `falsy`. I would just make the namespace optional as well